database/clickhouse/clickhouse.go

```diff
@@ -192,7 +192,7 @@ func (ch *ClickHouse) SetVersion(version int, dirty bool) error {
 		return err
 	}
 
-	query := "INSERT INTO " + ch.config.MigrationsTable + " (version, dirty, sequence) VALUES (?, ?, ?)"
+	query := "INSERT INTO `" + ch.config.MigrationsTable + "` (version, dirty, sequence) VALUES (?, ?, ?)"
```

Comment:
Note, there are still strings that would break this. e.g. if the table or db contained a `` ` ``
See also: https://github.com/golang/go/issues/18478

Reply:
Just merged another PR (https://github.com/golang-migrate/migrate/pull/857) that fixes this